apps/docs/content/guides/database/extensions/pgmq.mdx

```diff
@@ -0,0 +1,708 @@
+---
+id: 'pgmq'
+title: 'pgmq: Queues'
+description: 'pgmq: Managed queues in Postgres'
+---
+
+pgmq is a lightweight message queue built on Postgres.
+
+## Features
+
+- Lightweight - No background worker or external dependencies, just Postgres functions packaged in an extension
+- "exactly once" delivery of messages to a consumer within a visibility timeout
+- API parity with AWS SQS and RSMQ
+- Messages stay in the queue until explicitly removed
+- Messages can be archived, instead of deleted, for long-term retention and replayability
+
+## Enable the extension
+
+```sql
+create extension queue;
```

Comment:
is this a separate extension that's a dependency or did you mean `create schema`?

Reply:
good catch, that was meant to be `create schema queue`;
ty